docs/ios/platform/apple-pay.md

```diff
@@ -47,9 +47,8 @@ transaction does not go through Apple at all, instead, it goes through a Payment
 ## Using a Payment Processor Platform
 
 One of the fundamental parts of Apple Pay is the processing of payments. While it
-is possible to do this yourself, it requires significant knowledge of cryptography
-- as detailed in Apple’s [Payment Processing
```

Comment:
I think this was just because of text wrapping rather than an intentional list item